waiter/src/waiter/token.clj

```diff
@@ -601,6 +601,8 @@
       :get (let [{:strs [can-manage-as-user] :as request-params} (-> req ru/query-params-request :query-params)
                  include-deleted (utils/param-contains? request-params "include" "deleted")
                  show-metadata (utils/param-contains? request-params "include" "metadata")
+                 filter-maintenance (when (contains? request-params "maintenance")
+                                       (utils/request-flag request-params "maintenance"))
```

Comment:
Nitpick: The when is not necessary, it defaults to `false` when the parameter is not provided.

Reply:
I make a distinction between nil and false values because you can filter for "maintenance=true" or "maintenance=false" or not filter at all. I will split this into two separate variables

Reply:
Oops didn't realize the use of `nil` to indicate parameter being provided.